python/pyspark/mllib/util.py

```diff
@@ -169,6 +170,27 @@ def loadLabeledPoints(sc, path, minPartitions=None):
         minPartitions = minPartitions or min(sc.defaultParallelism, 2)
         return callMLlibFunc("loadLabeledPoints", sc, path, minPartitions)
 
+    @staticmethod
+    def appendBias(data):
+        """
+        Returns a new vector with `1.0` (bias) appended to
+        the end of the input vector.
+        """
+        vec = _convert_to_vector(data)
+        if isinstance(vec, SparseVector):
+            return sp.csc_matrix(np.append(vec.toArray(), 1.0))
```

Comment:
@jkbradley Thank you for response. I think if the returned value of `appendBias` should be `SparseVector`, there seems no space to use scipy.sparse because SparseVector can be constructed with given data without scipy.sparse. Is it correct?